pyproject.toml

```diff
@@ -636,8 +641,7 @@ ignore_errors = true
 
 [dependency-groups]
 dev = [
-  # TODO(potiuk): remove me when all providers are moved to new structure
-  "local-providers",
+  "apache-airflow",
```

Comment:
We need to ad dev dependency group from airflow and remvoe the "local-provider" one - this way pytest and others will be installed automatically when runnign `uv sync`